apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-examples/platform-multiselect-checkbox.component.html

```diff
@@ -0,0 +1,167 @@
+<h3>Template Driven Form</h3>
+<p>Checkbox can be pre-selected if value of checkbox matches any value in ngModel directive.</p>
+<fdp-form-group [multiLayout]="true">
+    <fdp-form-field [id]="'sirius'" zone="zLeft" rank="1">
+        <fdp-checkbox [name]="'sirius'" [label]="'Sirius'" [checkboxValue]="'sirius'" [(ngModel)]="field1"></fdp-checkbox>
```

Comment:
The idea behind formGroup/formField is that you should not repeat attributes which are set or should be set on the formField level. 

name = the same as ID or name if you set on formField Level

Reply:
moved common attribute to fdp-form-field. changed id and name value non-matching.